wesley-website/src/components/playground/PlaygroundNavbar.jsx

```
@@ -0,0 +1,113 @@
+    </UnstyledButton>
+  ));
+
+  const collectionLinks = outputFiles.map((file) => (
+    <a
+      href="#"
+      onClick={(event) => { event.preventDefault(); onSelectSidebarItem(file.file); }}
+      key={file.file}
+      className={cx(classes.collectionLink, { [classes.mainLinkActive]: activeFile === file.file })}
+      disabled={isTutorialActive && tutorialStepId !== 'sidebar-migrations'}
+      ref={tutorialRefs['sidebar-migrations']}
+    >
```

Biome (2.1.2)

[error] Use a button element instead of an a element.

Anchor elements should only be used for default sections or page navigation
Check this thorough explanation to better understand the context.

(lint/a11y/useValidAnchor)